lib/schemaUtils.js

```diff
@@ -406,7 +406,7 @@ module.exports = {
     let example = _.get(parameter, 'example'),
       examples = _.values(_.get(parameter, 'examples'));
 
-    if (example) {
+    if (example !== undefined && example !== null) {
```

Comment:
@LuisTejedaS JSON schema can also define `null` as type so we can remove `example !== null` check and only keep `undefined` one.

Reply:
Removing the example !== null is done, also added test for null type in bodies